### PR TITLE
Rewrite sym expr access method by visitor and close graph client connection in data loader

### PR DESCRIPTION
**Pull request by yixinglu**
as title.

### CODE REVIEW

File: src/visitor/RewriteSymExprVisitor.cpp [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+}
+
+void RewriteSymExprVisitor::visit(VersionedVariableExpression *expr) {
+    UNUSED(expr);
+    hasWrongType_ = true;
+}
+
+// container expression
+void RewriteSymExprVisitor::visit(ListExpression *expr) {
+    // FIXME(dutor)
```
Comment (CPWstatic) [inline src/visitor/RewriteSymExprVisitor.cpp:108]: Why `FIXME(dutor)` here?

**Review by CPWstatic — COMMENTED** [review]
LGTM.

File: src/visitor/RewriteSymExprVisitor.cpp (hunk 1 above)
Comment (yixinglu) [reply src/visitor/RewriteSymExprVisitor.cpp]: fixed

**Review by laura-ding — APPROVED** [review]

**Review by jievince — APPROVED** [review]
LGTM

**Pull request merged**